leveldb_os/expiry_os_test.cc

```diff
@@ -1349,5 +1350,170 @@ TEST(ExpiryManifestTester, Compact1)
 }   // Compact1
 
 
+struct sExpiryDBObject
+{
+    const char * m_Key;   // string key
+    const char * m_Value; // string value
+    int m_NowMinus;       // expiry time to set
+};
+
+
+class ExpiryDBTester
+{
+public:
+    ExpiryDBTester()
+        : m_Good(false), m_DB(NULL),
+          m_BaseTime(port::TimeUint64())
+    {
+        m_DBName = test::TmpDir() + "/expiry";
+
+        // clean up previous execution
+        leveldb::DestroyDB(m_DBName, m_Options);
+
+        m_Options.create_if_missing=true;
+        m_Options.error_if_exists=false;
+        m_Expiry=new leveldb::ExpiryModuleOS;
+        m_Options.expiry_module=m_Expiry;
+
+        OpenTestDB();
+    };
+
+    ~ExpiryDBTester()
+    {
+        // clean up
+        delete m_DB;
+        leveldb::DestroyDB(m_DBName, m_Options);
+    };
+
+    bool m_Good;
```

Comment:
Can add "protected:" above this line. While not necessary since this is a test class, it is still a goodness of OOP.